storage/engine/rocksdb_64bit.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2014 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+
+// +build amd64 arm64 arm64be ppc64 ppc64le mips64 mips64le s390x sparc64
+
+package engine
+
+const (
+	archMAXWIDTH    = 1 << 50
+	maxLen          = archMAXWIDTH - 1
```

Comment:
The constants here are defined slightly differently than in `rocksdb_32bit.go`. I have a mild preference to not defining `archMAXWIDTH` separately (unless that comes from somewhere else in which case it should be documented with a comment). And let's `s/maxLen/maxArrayLen/g` now that the scope of the constant has expanded to the package level.